model.go

```diff
@@ -1,63 +1,38 @@
 package kallax
 
 import (
-	"database/sql"
-	"database/sql/driver"
-
-	uuid "github.com/satori/go.uuid"
+	"github.com/src-d/go-kallax/behaviours"
+	"github.com/src-d/go-kallax/common"
 )
 
-type ID uuid.UUID
-
-func (id *ID) Scan(src interface{}) error {
-	return (*uuid.UUID)(id).Scan(src)
-}
-
-func (id ID) Value() (driver.Value, error) {
-	return uuid.UUID(id).Value()
-}
-
-var _ sql.Scanner = (*ID)(nil)
-var _ driver.Valuer = (*ID)(nil)
-
-func NewID() ID {
-	return ID(uuid.NewV4())
-}
-
-func (id ID) IsEmpty() bool {
-	return uuid.UUID(id) == uuid.Nil
-}
-
+// Model is the base type of the items that are stored
 type Model struct {
-	ID        ID
+	behaviours.Identificator
+	behaviours.TimestampDates
 	persisted bool
 	writable  bool
 }
 
+// NewModel creates and return a new Model
 func NewModel() Model {
-	return Model{
-		ID:        NewID(),
+	m := Model{
 		persisted: false,
 		writable:  true,
 	}
+	m.ID = common.NewID()
+	return m
 }
 
-func (m *Model) GetID() ID {
-	return m.ID
-}
-
-func (m *Model) SetID(id ID) {
-	m.ID = id
-}
-
+// IsPersisted returns whether this Model is new in the store or not.
 func (m *Model) IsPersisted() bool {
 	return m.persisted
 }
 
-func (m *Model) setPersisted(isPersisted bool) {
-	m.persisted = isPersisted
+func (m *Model) setPersisted() {
```

Comment:
`person.Persist()` should save the person into the db, what is not the purpose of this method.